### ISSUE TITLE
508-defect-2 [COGNITION]: Ch31 home, links must not rely on color

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22827]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Details

Links must not rely on color. Underlines are provided only on hover, which is not available to mobile users.

## Acceptance Criteria

- [ ] Links do not rely on color to convey that they can be interacted with 

## Proposed Solution (if known)

[As discussed in ticket #3556, these heading links should be underlined](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556).

## WCAG or Vendor Guidance (optional)

* [WCAG 1.4.1: Use of Color](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)

## Screenshots or Trace Logs

<img width="322" alt="Screen Shot 2021-04-07 at 11 06 44 AM" src="https://user-images.githubusercontent.com/14154792/113889684-75f56280-9791-11eb-8ea0-208fc1a6d4b5.png">

**Comment by micahchiang** [comment]
![Screen Shot 2021-09-29 at 1.54.48 PM.png](https://images.zenhubusercontent.com/5d68043b330f64311395e92b/06440b94-c227-4ef2-acf5-19cceed119ac)

@jason-gcio @joshkimux - I think we can close this ticket. Above screenshot is of production currently, where it looks like links at `/careers-employment/vocational-rehabilitation` include underlines.

**Comment by micahchiang** [comment]
...wow apologies for the gigantic screenshot. I'm not sure why it's so big haha.

**Comment by joshkimux** [comment]
Let's close it! 😄
